app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/WorkingAssert.kt

```diff
@@ -0,0 +1,9 @@
+package de.westnordost.streetcomplete.quests.opening_hours
+
+object Assert {
+    fun assert(conditionAssertedToBeTrue: Boolean) {
+        if(!conditionAssertedToBeTrue) {
+            throw AssertionError()
+        }
+    }
+}
```

Comment:
What's this? Are you looking for [require](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html) or [check](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/check.html)?
(See Preconditions.kt in the standard library)

Reply:
I initially used `assert()` then I discovered that by default it is disabled.

And later I started wondering how to handle this in production code - just abort by returning null? Abort by throwing an exception and catching it? Abort by throwing an exception, catch it and log. Refuse to catch exception and crash only in the debug mode?

All of asserts are for testing whatever assumptions made by subsequent code are true, during development asserts were already useful and will be probably useful in case of future expansions of